doc/source/installation/yum_repo.rst

```diff
@@ -51,24 +51,30 @@ Installing |Percona Server| from Percona ``yum`` repository
 
    .. code-block:: bash
 
-     yum install http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
+      $ yum install http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
 
-   You should see some output such as the following: 
+   .. admonition:: Output example
+
+      .. code-block:: guess
+
+	 Retrieving http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
+	 Preparing...                ########################################### [100%]
+         1:percona-release        ########################################### [100%]
+
+   To install |Percona Server| with SELinux policies, install the :program:`Percona-Server-selinux-*.noarch.rpm` package instead:
```

Comment:
why "instead"?

Reply:
Above in this step yum install shows a non-selinux package

Reply:
Both packages are needed for SELinux to work. SElinux package doesn't replace the server package.

Reply:
Ready for review